### PR TITLE
Add `cf files` documentation;

### PR DESCRIPTION
**Pull request by rogeruiz**
This patch adds documentation for `cf files` under Databases as it seems
to be the most useful when paired with accessing `psql` on the server.

### CODE REVIEW

**Comment by rogeruiz** [comment]
cc: @afeld @pkarman Let me know if that language works for using `cf files`. I can also move it to a more general place as well, or maybe link to it from the [General page](https://docs.cloud.gov/apps/deployment/) as well? 💭

File: content/apps/databases.md [hunk 1]
```diff
@@ -62,3 +62,36 @@ To access a service database, use the [cf-ssh]({{< relref "getting-started/cf-ss
...
+```sh
+cf files <APP_NAME>
+```
+
+To save a file to your local machine, you can redirect the output from `stout`
+to a file using the `>` character and then a file path on your local machine.
+Here's an example:
+
+```sh
+cf files my-app-ssh app/path/to/database_dump.sql > ./database_dump.sql
```
Comment (jmcarp) [inline content/apps/databases.md]: At least on cf cli v6.15, this adds some extra lines to the file, e.g.

```
Getting files for app eregs-web in org eregs-nc / space dev as joshua.carp@gsa.gov...
OK

<real content starts here>
```

How about something like `cf files my-app-ssh path/to/thing | tail -n +3 > out.sql`?

Comment (rogeruiz) [inline content/apps/databases.md]: yeah. That's a great suggestion @jmcarp Thanks. I'll add it in.

File: content/apps/databases.md [hunk 2]
```diff
@@ -93,5 +93,5 @@ to a file using the `>` character and then a file path on your local machine.
 Here's an example:
 
 ```sh
-cf files my-app-ssh app/path/to/database_dump.sql > ./database_dump.sql
+cf files my-app-ssh app/path/to/database_dump.sql | tail -n +4 > ./database_dump.sql
```
Comment (rogeruiz) [inline content/apps/databases.md:96]: Ignoring the first few lines so files don't need to be edited after downloading. Thanks @jmcarp! 🎉

**Comment by pkarman** [comment]
tested and works wonderfully.

**Pull request merged**